src/lib/viewers/controls/settings/SettingsMenu.tsx

```diff
@@ -37,27 +22,18 @@ export default function SettingsMenu({ children, className, name }: Props): JSX.
         }
     }, [isActive, setActiveRect]);
 
-    React.useEffect(() => {
-        if (activeItem && isActive) {
-            activeItem.focus();
-        }
-    }, [activeItem, isActive]);
+    if (!isActive) {
```

Comment:
Yeah it's needed because `SettingsFlyout` doesn't play nice with multiple instances sharing the same `activeRect`. We can look to revert this with a `Flyout` mixin like we discussed earlier